content/en/getting_started/logs/_index.md

```diff
@@ -6,6 +6,9 @@ further_reading:
       tag: 'Learning Center'
       text: 'Introduction to Logs in Datadog'
     - link: '/logs/log_collection/'
+    - link: /tagging/unified_service_tagging
+      tag: 'Documentation'
+      text: 'Learn how to configure unified service tagging for logs'
```

Comment:
Should we say `for logs` or insists on the cross-product benefit for all products?

Something like `learn how to configure unified service tagging across all your data types`